terraform/lambda/README.md

```diff
@@ -0,0 +1,11 @@
+Example sandbox.tfvars file:
+
+```
+aws_account_id                    = "<your_account_id>"
+aws_access_key_id                 = "<your_access_key_id>"
+aws_secret_access_key             = "<your_secret_access_key>"
+domain                            = "sdp-sandbox"
+ecr_repository                    = "keh-tech-audit-tool-lambda-api" # ECR repository name where the lambda image is stored
+tech_audit_data_bucket_name       = "keh-tech-audit-tool" # S3 bucket name where the tech audit data is stored
```

Comment:
Do these names start with keh or sdp-dev - your example_tfvars is sdp-dev ?

Reply:
We don't need this README as we have the example_tfvars, so I removed it.